recipe/activate.sh

```diff
@@ -13,6 +13,9 @@ export CONDA_RUST_HOST=@rust_arch_env_build@
 export CONDA_RUST_TARGET=@rust_arch_env@
 export PKG_CONFIG_PATH_@rust_arch_env_build@=${CONDA_PREFIX}/lib/pkgconfig
 export PKG_CONFIG_PATH_@rust_arch_env@=${PREFIX:-${CONDA_PREFIX}}/lib/pkgconfig
+export CONDA_RUST_HOST_LOWER=$(echo ${CONDA_RUST_HOST} | tr '[:upper:]' '[:lower:]')
+declare "CC_${CONDA_RUST_HOST_LOWER}"="${CC_FOR_BUILD:-${CONDA_PREFIX}/bin/@rust_default_cc_build@}"
+export "CC_${CONDA_RUST_HOST_LOWER}"
```

Comment:
Thanks for the PR

We can calculate `CONDA_RUST_HOST_LOWER` in build.sh and do
```suggestion
export @CONDA_RUST_HOST_LOWER@="${CC_FOR_BUILD:-${CONDA_PREFIX}/bin/@rust_default_cc_build@}"
```
which makes the package not require `tr` or the unusual declare statement which I'm not sure is POSIX compatible.

Reply:
Thanks, good idea. I think I have it done right but please make sure.